bin/set-up-current-account.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -euo pipefail
+
+# Name the account based on the current account alias
+ACCOUNT_ALIAS="$(./bin/current-account-alias.sh)"
+ACCOUNT_ID="$(./bin/current-account-id.sh)"
+REGION="$(./bin/current-region.sh)"
+BACKEND_CONFIG_FILE="$ACCOUNT_ALIAS.s3.tfbackend"
```

Comment:
I don't think `BACKEND_CONFIG_FILE` variable is being used. Seems like its being handled on line 23 of `bin/create-tfbackend.sh` which uses the `BACKEND_CONFIG_NAME` passed as an argument on line 62 of this file.